go/types/blob.go

```diff
@@ -295,6 +295,7 @@ func readBlob(r io.Reader, vrw ValueReadWriter) Blob {
 	mtChan := make(chan chan metaTuple, runtime.NumCPU())
 
 	makeChunk := func() {
+		rv.Reset()
```

Comment:
FWIW, Compressing the rolling hash using the snappy variants, increased the cascade rate (unsurprisingly), which is another part of why I think it makes sense to hold off on that.

I suspect that when we can apply bulk updates to ptrees (and have lazily decoding), the win will be big -- because it increases the value density more than 2x. IOW, we trade a modestly larger cascade rate for dramatically lower read IO when doing bulk edits to ptrees. IOW, I'm waiting until it's an evident win on the work loads we have.